app/src/main/res/layout/fragment_edit_profile.xml

```diff
@@ -25,190 +25,151 @@
-        android:importantForAutofill="no"
-        android:inputType="text" />
-
-        <TextView
-            android:id="@+id/tvLocationLabel"
+        <LinearLayout
             android:layout_width="wrap_content"
             android:layout_height="wrap_content"
-            android:text="@string/location_profile_label"
-            android:textStyle="bold"
-            android:textColor="@color/white"
-            app:layout_constraintStart_toStartOf="@+id/editLocationField"
-            app:layout_constraintTop_toBottomOf="@+id/editBio" />
-
-    <EditText
-        android:id="@+id/editLocationField"
-        android:layout_width="wrap_content"
-        android:layout_height="wrap_content"
-        android:background="@android:color/transparent"
-        android:hint="@string/enter_location_here_hint"
-        android:minHeight="48dp"
-        android:textColor="@color/purple_200"
-        android:textColorHint="@color/hint_color"
-        app:layout_constraintStart_toStartOf="@+id/tvPasswordLabel"
-        app:layout_constraintTop_toBottomOf="@+id/tvLocationLabel"
-        android:inputType="text"
-        android:autofillHints="" />
-
-        <TextView
-            android:id="@+id/tvPasswordLabel"
-            android:layout_width="wrap_content"
-            android:layout_height="wrap_content"
-            android:text="@string/password_profile_label"
-            android:textStyle="bold"
-            android:textColor="@color/white"
-            app:layout_constraintStart_toStartOf="@+id/editPasswordLayout"
-            app:layout_constraintTop_toBottomOf="@+id/editLocationField" />
-
-        <com.google.android.material.textfield.TextInputLayout
-            android:id="@+id/editPasswordLayout"
-            android:layout_width="150dp"
-            android:layout_height="50dp"
-            android:background="@android:color/transparent"
-            app:layout_constraintEnd_toEndOf="parent"
-            app:layout_constraintHorizontal_bias="0.045"
-            app:layout_constraintStart_toStartOf="parent"
-            app:layout_constraintTop_toBottomOf="@+id/tvPasswordLabel"
-            android:textColor="@color/purple_200"
-            app:passwordToggleEnabled="true">
-
-            <com.google.android.material.textfield.TextInputEditText
-                android:id="@+id/editPasswordField"
-                android:layout_width="match_parent"
-                android:layout_height="match_parent"
-                android:minWidth="45dp"
-                android:background="@android:color/transparent"
-                android:textColor="@color/purple_200"
-                android:ems="30"
-                android:inputType="textPassword|textNoSuggestions" />
-        </com.google.android.material.textfield.TextInputLayout>
+            android:layout_below="@id/editProfileFields"
+            android:layout_centerHorizontal="true"
+            android:layout_marginTop="15dp">
+
+            <Button
```

Comment:
Creating two separate buttons for Save/Cancel were such a great design choice, Vivian! With removal of the password field, that really fills the space in